packages/bitcore-client/bin/multi-sig/wallet-derive-address

```diff
@@ -64,23 +64,39 @@ function getAllAddresses(publicKeyRing, numberOfAddresses, isChange, m, network)
   return addresses;
 }
 
+async function readFromPrompt() {
```

Comment:
I can't comment on the correct lines.  Can you remove the `console.log` on line 62.  And the comments on line 45 to 50.  This afternoon a way to configure this to use configurable derivation strategies.